<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browseButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePathLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select the file to encrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select the folder to save encrypted file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveFolderPathLineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFolderBrowseButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255)</string>
    </property>
    <property name="text">
     <string>Encryptor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionAssetLocation">
   <property name="text">
    <string>Asset Location</string>
   </property>
  </action>
  <action name="actionSaveLocation">
   <property name="text">
    <string>Save Location</string>
   </property>
  </action>
  <action name="actionSolutionAssetLocation">
   <property name="text">
    <string>Solution Asset Location</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
